backend/app.py

```
@@ -1006,7 +1009,11 @@ async def get_divisions():
 
 
 @app.get("/api/teams")
-async def get_teams(match_type_id: int | None = None, age_group_id: int | None = None):
+async def get_teams(
+    current_user: dict[str, Any] = Depends(get_current_user_required),
```

Ruff (0.14.1)

Unused function argument: `current_user`

(ARG001)